target/scripts/startup/setup-stack.sh

```diff
@@ -960,6 +960,22 @@ function _setup_security_stack
       sed -i "s|\$final_spam_destiny.*=.*$|\$final_spam_destiny = D_BOUNCE;|g" /etc/amavis/conf.d/49-docker-mailserver
       sed -i "s|\$final_bad_header_destiny.*=.*$|\$final_bad_header_destiny = D_BOUNCE;|g" /etc/amavis/conf.d/49-docker-mailserver
     fi
+
+    if [[ ${ENABLE_SPAMASSASSIN_KAM} -eq 1 ]]
+    then
+      _notify 'inf' 'Configuring Spamassassin KAM'
+      local SPAMASSASSIN_KAM_CRON_FILE=/etc/cron.daily/10-spamassassin_kam
```

Comment:
```suggestion
      local SPAMASSASSIN_KAM_CRON_FILE=/etc/cron.daily/spamassassin_kam
```

The other files in that directory don't use a number prefix. Unless there is a reason for that, we should keep it uniform.

Reply:
There is a reason: this should run before the normal Spamassassin cron script runs. When I tend to uniform cron-handling, I will see that all scripts get a number prefix.

Reply:
`crond` does not care about naming and will run all `@daily` jobs in parallel (with some ms delay of course). It will not run the jobs sequential and wait for each to be finished). That's why no script in that directory uses the naming scheme.

> This should run before the normal Spamassassin cron script

What's the exact reason? I guess it's because SA cron reloads the service? Two suggestions to solve this:

1. Add reload function to the KAM cronjob (stolen from `/etc/cron.daily/spamassassin`):	
```
# Tell a running spamd to reload its configs and rules.
reload() {
    # Reload
    if which invoke-rc.d >/dev/null 2>&1; then
        invoke-rc.d --quiet spamassassin status > /dev/null && \
          /etc/init.d/spamassassin reload > /dev/null
    else
        /etc/init.d/spamassassin reload > /dev/null
    fi
    if [ -d /etc/spamassassin/sa-update-hooks.d ]; then
        run-parts --lsbsysinit /etc/spamassassin/sa-update-hooks.d
    fi
}
```

2. Move `/etc/cron.daily/spamassassin` to `/etc/cron.d/spamassassin` and source it in the KAM cronjob after the KAM update.

The second solution seems more elegant on first sight to me, but then I realized, the SA cronjob will only reload SA, if there where SA updates. So if there are KAM updates, but no SA updates, this will not work reliable.

Reply:
I didn't know that cron executes "in parallel" :D Nice to know! I think it is not strictly required that one runs after another because there is at most a delay of one day (if KAM updates are received after the normal SA updates run, the next day, SA will also update the KAM signatures). I will renamed the file accordingly.

Reply:
I don't know how often the rules are updated and if one day delay is "acceptable".
But by adding `&& /etc/init.d/spamassassin reload` we should be on the safe side?